### ISSUE TITLE
Test with my own image

### ISSUE THREAD

**Issue by yixionghuo** [lucastabelini/LaneATT#117]
Thanks for your great work.
I have a question. I have tested my own data, the program results show high accuracy( {'Accuracy': 0.9719642857, 'FP': 0.0, 'FN': 0.0, 'FPS': 1000.0}). But when I output the result, the effect is very poor. I want to know why.
![下载](https://user-images.githubusercontent.com/66975556/170662538-39f25b18-4bdb-424e-86c5-54aaee4beeae.jpeg)

**Comment by yixionghuo** [comment]
I also want to ask why there are many lanes marked in my training data set, but only one lane can be predicted in test set.

**Comment by zhangzhongyuan1996** [comment]
找到方法了吗兄弟
